README.md

```diff
@@ -9,7 +9,7 @@
 
 ### Deployment
 
-Wesite:[Heroku](https://secret-plains-32456.herokuapp.com/)
+Wesite: [Heroku](https://secret-plains-32456.herokuapp.com/)
 
 
 ## Getting Started
```

Comment:
The getting started section of your README is not explicit enough. You have to describe how someone can have your project cloned on his/her local machine.